clouddriver-lambda/src/main/java/com/netflix/spinnaker/clouddriver/lambda/service/LambdaService.java

```diff
@@ -315,35 +274,10 @@ private List<String> getTargetGroupNames(String functionName) {
 
     } catch (ResourceNotFoundException e) {
       // ignore the exception.
+    } catch (NullPointerException e) {
```

Comment:
does it make sense to log anything here?

Reply:
It's not been logged in the past... but I"ll add an info level log.  More logs are useful :)